src/Post.php

```diff
@@ -686,6 +686,9 @@ public static function render_navigation_block(array $attributes): string
         // For parent pages, get the previous and next siblings in the menu order
         $output = '';
         $siblings = array_filter($menu_items, function ($item) use ($nav_menu_item) {
+            if (!$item) {
```

Comment:
Can you update this check as -
```
         return $item !== null
	            && $nav_menu_item !== null
	            && $item->menu_item_parent === $nav_menu_item->menu_item_parent;
```
This makes the check more explicit and improves readability.